namespace-to-cjs.rb

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env ruby
+
+require 'find'
+require 'fileutils'
+require 'pry'
+
+
+
+# Constants
+
+CLASS_EXPRESSION = /^class\s(?:@)?([A-Z][\w\.]*)/
+DEPENDENCY_EXPRESSION = /^(?!(\s\*|#)).*(?:new|extends)\s(Brainstem[\w\.]*)/
+
+REQUIRE_TEMPLATE = "%{short_name} = require('%{path}')"
+
+# Globals
+
+$class_map = {}
+$files = []
+
+
+
+# Helpers
+
+def enumerate_files
+  Find.find('vendor/assets/javascripts/') do |path|
```

Comment:
Yes, all of this hardcoded stuff needs to be moved to argument parameters. Ultimately this script should be in it's own repo with some dummy code and tests. As we find more gotchas we can add more dummy code to test.

This needs to be removed before this branch is meged regardless.